pkg/serverless/registration/extension_api_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const registerExtensionTimeout = 10 * time.Millisecond
+const registerExtensionTimeout = 100 * time.Millisecond
```

Comment:
This test was failing because the client timeout was too low.

Solution is to increase timeout.